api/models/Post.js

```diff
@@ -491,10 +492,10 @@ module.exports = bookshelf.Model.extend(Object.assign({
 
   fixTypedPosts: () =>
     bookshelf.transaction(transacting =>
-      Tag.where('name', 'in', ['request', 'offer', 'intention'])
+      Tag.where('name', 'in', ['request', 'offer', 'resource', 'intention'])
```

Comment:
huh, I wonder what intention is/was? i also wonder if this is old code using Tags for type instead of the post type column